### ISSUE TITLE
how do you train MobileNet?

### ISSUE THREAD

**Issue by laochonlam** [Xilinx/brevitas#387]
Hi @volcacius ,

Thanks for releasing the code and your effort. I like this project!

My goal is to compile the Brevitas MobileNet model with my own dataset trained then deploy it to FINN. However, I couldn't find out how you train/retrain the MobileNet model but this issue (https://github.com/Xilinx/brevitas/issues/42).

I am wondering if you still plan to release the MobileNet training script? Also, I am not quite sure If I understood your reply to [that issue](https://github.com/Xilinx/brevitas/issues/42): 

It seems that you used the Brevitas MobileNet to fine-tune the pre-trained floating-point MobileNet weights got from [here](https://github.com/osmr/imgclsmob/tree/master/pytorch)? If so, I assume the learning rate or other training parameters should be quite different from normal training. It would be great if I can get any tutorial from your side. 

Sincerely,
ChonLam

**Comment by Ba1tu3han** [comment]
Hello,

Did you find how to train the model?
